src/oatmeal-chocolate-chip-cookies.md

```diff
@@ -0,0 +1,24 @@
+# Oatmeal Chocolate Chip Cookies
+
+An unfortunately uncommon style of cookie that is absolutely delicious.
+
+## Ingredients
+
+- 1 &half; c flour
```

Comment:
I assume `c` refers to cups?

Reply:
Yes, the source uses those old-style abbreviations.